tests/api_connexion/schemas/test_task_instance_schema.py

```diff
@@ -236,6 +236,22 @@ def test_success(self):
         }
         assert expected_result == result
 
+    def test_dry_run_is_optional(self):
+        data = self.current_input.copy()
+        data.pop("dry_run")
+        result = set_task_instance_state_form.load(self.current_input)
+        expected_result = {
+            "dry_run": True,
+            "execution_date": dt.datetime(2020, 1, 1, 0, 0, tzinfo=dt.timezone(dt.timedelta(0), "+0000")),
+            "include_downstream": True,
+            "include_future": True,
+            "include_past": True,
+            "include_upstream": True,
+            "new_state": "failed",
+            "task_id": "print_the_context",
+        }
+        assert expected_result == result
+    
```

Comment:
Extra spaces here, it should be removed for pass [Static Checks](https://github.com/apache/airflow/blob/main/STATIC_CODE_CHECKS.rst#static-code-checks)

Reply:
May want to setup pre-commit hooks - https://github.com/apache/airflow/blob/main/CONTRIBUTORS_QUICK_START.rst#id12